submissions/YuliiaDyka/pop-up/index.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Popup</title>
+</head>
+    <body>
+        <header class="header">
+            <div class="logo">
+                <a class="logo__link" href="https://github.com/Yuliiadd/kottans-frontend"><img class="logo__img" src="img/logo.png" alt="Go to homepage"></a>
+            </div>
+            <nav class="nav">
+                <ul class="navigation__list" >
+                    <li class="nav_item"><a class="link" href="#">Mail</a></li>
+                    <li class="nav_item"><a class="link" href="#">Images</a></li>
+                    <li class="nav_item">
+                        <div class="ch_box popup">
+                            <label for="checkbox">
+                            <img class="nav_img check" src="img/popup-button.png">
+                            </label>
+                            <input type="checkbox" id="checkbox" class="checkbox">
+                            <div class="context__menu">
```

Comment:
Please increase the popup items size
![image](https://user-images.githubusercontent.com/16196199/187025982-c6689f7c-236c-4fe0-9eaf-fcefc7064c2c.png)